.github/workflows/check_suite.yaml

```diff
@@ -0,0 +1,27 @@
+name: Check Enforcer
+
+on:
+  check_suite:
+    types: [completed]
+
+jobs:
+  check-enforcer:
+    name: Evaluate Check Suite
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Run
+        env:
+          GITHUB_PAYLOAD: ${{ toJson(github.event) }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "Sparse checkout azure-sdk-tools..."
```

Comment:
Lets see what this looks like without needing to do a sparse checkout. The normal checkout I believe is already shallow by default. However if we can move this to be an action outside of the repo maybe we don't need to explicitly checkout at all because github will handle it.

Reply:
In my initial testing, using `uses` without the actions normal checkout step (i.e. trying to use sparse) didn't work due to missing files. So I believe it is required and does not work the way devops does with its pre-compile step.